easybuild/easyconfigs/p/Phantompeakqualtools/Phantompeakqualtools-1.2.2-foss-2021b.eb

```
@@ -0,0 +1,54 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'Phantompeakqualtools'
+version = '1.2.2'
+
+homepage = "https://github.com/kundajelab/phantompeakqualtools"
+
+description = """ It computes informative enrichment and quality measures for ChIP-seq/DNase-seq/FAIRE-seq/MNase-seq data."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+
+# Download manually from https://github.com/kundajelab/phantompeakqualtools/releases
+sources = [SOURCELOWER_TAR_GZ]
```

The tarball seems to be publicly available, so there's no need for a manual download here:
```suggestion
source_urls = ['https://github.com/kundajelab/%(namelower)s/archive/refs/tags/']
sources = ['%(version)s.tar.gz']
```